app/models/health/patient.rb

```diff
@@ -399,9 +399,17 @@ def self.cp_2_engagement(on) # rubocop:disable Naming/MethodParameterName
     end
 
     scope :needs_intake, ->(on: Date.current.end_of_month) do
+      intake_due(on: on).or(intake_overdue(on: on))
+    end
+
+    scope :intake_due, ->(on: Date.current.end_of_month) do
```

Comment:
the `on - 30.days` may be a problem for February...  maybe that should be `on.beginning_of_month .. on`

Reply:
I think MassHealth specified the 30 and 60 day rolling windows, but worth keeping in mind.